### PR TITLE
Add banner on landing page linking to idseq public site

### PR DESCRIPTION
**Pull request by lvreynoso**
# Description

Adds a banner at the top of the page linking to https://public.idseq.net/ to redirect people who may be looking for our response to the nCov-2019 virus.

Screen Shot 2020-03-05 at 10 07 30 AM

### CODE REVIEW

**Comment by jshoe** [comment]
I'll point out that the current video bulletin is still active, so it'll look a little different

File: app/assets/src/components/views/Landing.jsx [hunk 1]
```diff
@@ -125,6 +131,39 @@ class Landing extends React.Component {
...
+                  Learn how researchers in Cambodia used IDseq to sequence
+                  SARS-CoV-2
+                </ExternalLink>
+              </span>
+            }
+          />
+        </div>
+      );
+    } else {
+      publicSiteBanner = <Divider />;
```
Comment (jshoe) [inline app/assets/src/components/views/Landing.jsx]: The `else` Divider isn't really needed, right?

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -25,6 +25,19 @@ def landing
           @show_bulletin = true
         end
       end
+
+      @show_public_site = false
+      if get_app_config(AppConfig::SHOW_LANDING_PUBLIC_SITE_BANNER) == "1"
+        @show_public_site = true
+      else
+        time_zone = ActiveSupport::TimeZone.new("Pacific Time (US & Canada)")
```
Comment (jshoe) [inline app/controllers/home_controller.rb]: I don't think you really need the time check for this one ;)

**Comment by lvreynoso** [comment]
Going to do this as a "release fix" instead

**Pull request closed without merging**